posts/views.py

```diff
@@ -86,6 +91,39 @@ def PostRepostView(request, pk=None):
     return redirect('posts:post', pk=repost.pk)
 
 
+def ParseContent(content):
```

Comment:
You're right. Sorry. Will do refactoring tomorrow after merging all pull requests for this sprint.